server/controllers/class/putAssignment.ts

```diff
@@ -0,0 +1,16 @@
+import { Request, Response, NextFunction } from 'express';
+import { putAssignmentQuery } from '../../queries/class';
+
+const putAssignment = async (req:Request, res:Response, next:NextFunction) => {
+  try {
+    const { assignmentId } = req.params;
+    const { title, description } = req.body;
+
+    const [, data]:any = await putAssignmentQuery(assignmentId, title, description);
```

Comment:
Try to throw an error when the query returns nothing but null, it happened to me and crashed the server!
Also, make sure that the data is sent in the body, otherwise, throw an error!